docs/getting-started/proofs-and-verification/gosdk/tx.md

```diff
@@ -3,3 +3,36 @@ id: tx
 title: Proof of Existence of a Transaction
 ---
 
+### Transaction proof example
+Attached simple code snippet to show how to use ledger API for transaction existences proof:
```

Comment:
Below is a simple code snippet that shows how to use the ledger API in order to generate a transaction existence proof: